src/use-formbit.ts

```diff
@@ -107,7 +107,13 @@ export default <Values extends InitialValues>({
     } = {},
     action
   ) => {
-    const newUUID = uuidv4()
+    const newUUID = (function getUUID() {
```

Comment:
since the function getUUID is reused in multiple places, can we move it to call it up?

Reply:
I'd prefer to keep the auto-invoking function there because of the readability